Lib/socket.py

```diff
@@ -237,26 +237,35 @@ def makefile(self, mode="r", buffering=None, *,
             rawmode += "r"
         if writing:
             rawmode += "w"
+
```

Comment:
A core developer will likely request that you remove the blank lines that have been added in this file.

Reply:
Thanks for the review :)
I found this method hard to read so dividing it into several code blocks helped my understanding of it. I can either remove the blank lines or add extra comments if necessary.